models/us_cms_open_payments/code/register_metadata.py

```
@@ -0,0 +1,400 @@
+            category="organization", slug=meta.ORGANIZATION_SLUG, env=env
+        )["id"]
+        self.tags = [self._tag(aliases) for aliases in meta.TAG_SLUGS]
+
+    def _tag(self, aliases: tuple[str, ...]) -> str:
+        for slug in aliases:
+            try:
+                return server.lookup_id(
+                    category="tag", slug=slug, env=self.env
+                )["id"]
+            except Exception:
+                continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)